modules/ROOT/pages/prereq-env.adoc

```diff
@@ -0,0 +1,42 @@
+= Environment Limitations
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Adhere to the following recommendations and limitations when setting up your environment and deploying apps:
+
+* Environment:
+
+** Production: 6 nodes of 8 cores and 32GB 
+** Maximum number of servers: 500
+** Maximum number of servers per server group: 20
+** Maximum servers per cluster: 8
+
+* Sizing
+
+** Runtime Manager:
+
+*** Applications per Environment 
```

Comment:
I'd say, merging with the text below as well:
```
*** Mule applications under management:
**** Recommended: around 200
**** Maximum: 500

*** Mule runtimes under management:
**** Maximum: 800 single Mule runtimes (or 400 Mule runtime clusters, consisting of 2 Mule runtimes per cluster)

*** Mule applications per Mule runtime:
***** Production: Maximum of 50 apps 
***** Development: Maximum of 100 apps
***** Recommended: Fewer than 25 apps
```

Reply:
Made updates.